features/div.yml

```
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <div>
-description: The `<div>` element TODO.
+description: The `<div>` HTML element is the generic container for flow content.
```

I'm guessing few developers could give a clear definition of "flow content." Feels a bit like spec speak. What do you think of one of these?

Just being more casual:

```suggestion
description: The `<div>` HTML element is the generic container for content.
```

Or defining the term a bit:

```suggestion
description: The `<div>` HTML element is the generic container for flow content (text and most elements typically found in `<body>`).
```

---

I went with the more generic since people use div's for more than just text.